specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json

```diff
@@ -357,6 +357,10 @@
             }
           ]
         },
+       "rankerType": {
+            "type": "string",
+            "description": "Optional field. Set to 'QuestionOnly' for using a question only Ranker.",
```

Comment:
Please remove trailing comma.

Reply:
@jhendrixMSFT  fixed this. Yes, this change is live.